.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules/
+rest.md
```

Comment:
## Changes Required!
⚠️  Your .gitignore is not listing `test.md`; instead, you have a typo where it says `rest.md`. Kindly correct this error to properly ignore `test.md` and comply with all the requirements!